contracts/src/bridge/SequencerInbox.sol

```diff
@@ -23,11 +23,9 @@ import {MAX_DATA_SIZE} from "../libraries/Constants.sol";
  * sequencer within a time limit they can be force included into the rollup inbox by anyone.
  */
 contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox {
-    bytes32[] public override inboxAccs;
     uint256 public totalDelayedMessagesRead;
 
     IBridge public delayedBridge;
```

Comment:
Maybe we should rename this to just `bridge` if it also stores sequencer batches